kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div class="task-panel" :class="{ 'task-panel-sm': windowIsSmall }">
+  <div class="task-panel" :class="{ 'task-panel-sm': windowWidth<600 }">
```

Comment:
`windowIsSmall` is only working with `KResponsiveWindow`, not with `userKResponsiveWIndow`.
But this issue was with only few pages

Reply:
Yes, it does work. Thank You